pyccl/neutrinos.py

```diff
@@ -75,27 +83,46 @@ def nu_masses(*, Omega_nu_h2, mass_split, T_CMB=None):
     Returns:
         float or array-like: Neutrino mass(es) corresponding to this Omeganuh2
     """
-    status = 0
-
     if T_CMB is not None:
         warnings.warn("T_CMB is deprecated as an argument of `nu_masses.",
                       CCLDeprecationWarning)
+    return get_neutrino_masses(m_nu=93.14*Omega_nu_h2, mass_split=mass_split)
 
-    if mass_split not in neutrino_mass_splits.keys():
-        raise ValueError(
-            "'%s' is not a valid species type. "
-            "Available options are: %s"
-            % (mass_split, neutrino_mass_splits.keys()))
-
-    # Call function
-    if mass_split in ['normal', 'inverted', 'equal']:
-        mnu, status = lib.nu_masses_vec(
-            Omega_nu_h2, neutrino_mass_splits[mass_split], 3, status)
-    elif mass_split in ['sum', 'single']:
-        mnu, status = lib.nu_masses_vec(
-            Omega_nu_h2, neutrino_mass_splits[mass_split], 1, status)
-        mnu = mnu[0]
 
-    # Check status and return
-    check(status)
-    return mnu
+def get_neutrino_masses(*, m_nu, mass_split):
```

Comment:
TODO: Maybe make this a private func to avoid cluttering the namespace. But it's a nice func to expose so idk.